Dockerfile

```diff
@@ -13,35 +13,38 @@
 # function akamai { if [[ `docker ps | grep akamai-cli$ | wc -l` -eq 1 ]]; then docker exec -it akamai-cli akamai $@; elif docker start akamai-cli > /dev/null 2>&1 && sleep 3 && docker exec -it akamai-cli akamai $@; then return 0; else echo "Creating new docker container" docker create -it -v $HOME/.edgerc:/root/.edgerc --name akamai-cli akamai/cli > /dev/null 2>&1 && akamai $@; fi; }
 FROM alpine 
 ARG SOURCE_BRANCH=master
-ENV SOURCE_BRANCH="$SOURCE_BRANCH" GOROOT=/usr/lib/go GOPATH=/gopath GOBIN=/gopath/bin AKAMAI_CLI_HOME=/cli
-RUN mkdir /cli && \
+ARG AKAMAI_CLI_PACKAGE_REPO=https://developer.akamai.com/cli/package-list.json
```

Comment:
Might want to name this `AKAMAI_CLI_PACKAGE_LIST` instead?

Reply:
I've just re-used the same name as the env var used in command_search.go:72 to keep some coherence. Should we align both to AKAMAI_CLI_PACKAGE_LIST ?